.github/workflows/build_ubuntu.yml

```diff
@@ -0,0 +1,240 @@
+name: Build Ubuntu
```

Comment:
This is a bit confusing compared to the `build_linux` workflow, some comments (and renaming of workflows/jobs/others), would be helpful in understanding the different things tested. Is this test specific to ubuntu as in "check that alt-ergo can build using the versions of ocaml and deps that are available on ubuntu (as opposed to generic linux)" ?

Reply:
`build_linux` is now `build_docker` it should be clearer